### PR TITLE
[JENKINS-73695] BUG: Dashboard shows white space on certain width space >900px

### PR DESCRIPTION
**Pull request by scherler**
Problem:
image
After with the pr:
image

See [JENKINS-73695](https://issues.jenkins.io/browse/JENKINS-73695).

### Proposed changelog entries

- human-readable text

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@janfaracik @timja @basil 

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

### CODE REVIEW

**Comment by scherler** [comment]
@janfaracik @timja There is a different approach to solve the underlying problem which would mean to change `war/src/main/scss/base/_breakpoints.scss` from `900` to `1020` however this PR is more robust since the breakpoint is a bit more based on try and error and in case the table is extended by some plugins the underlying bug would be present.

File: war/src/main/scss/abstracts/_theme.scss
Comment (timja) [inline war/src/main/scss/abstracts/_theme.scss:270]: typo?
```suggestion
  --side-panel-width: 340px;
```

File: core/src/main/resources/lib/hudson/projectView.jelly
Comment (janfaracik) [reply core/src/main/resources/lib/hudson/projectView.jelly:70]: I would agree with this ^

`app-project-status-table` would make it clear that this is an internal class (and could change).

**Comment by janfaracik** [comment]
Tried it and works fine 👍 

---

One for the future but would be good to have the table be properly responsive in the future, shuffling/showing/hiding elements as necessary to fit the screen.

File: core/src/main/resources/lib/hudson/projectView.jelly
Comment (scherler) [reply core/src/main/resources/lib/hudson/projectView.jelly:70]: Thanks for explaining, I will change this

**Comment by janfaracik** [comment]
> @janfaracik yeah I agree, but in the end I did not wanted to start this since I am not sure about the extensibility of the columns and whether or not we want to combine the media query with using `columnExtensions` I guess we would need a discussion to which columns we want `drop` and which to `keep`

Definitely - will definitely be a big job.

I'm conducting some research on that topic now actually - https://forms.gle/51uqa5yyPvknDETb9 - if you'd be free for filling it in :) Should take a couple minutes at most.

**Review by timja — APPROVED** [review]
Thanks!

**Review by Artmorse — APPROVED** [review]

**Comment by timja** [comment]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Comment by strangelookingnerd** [comment]
@Artmorse @janfaracik 

I suspect this change to have introduced a small regression, as there seems to be a "useless" horizontal scrollbar in the overviews:

grafik

This may have gone unnoticed as the scrollbar disapears when the list / table loses focus.

**Comment by scherler** [comment]
I cannot reproduce it in Chrome (the focus should be on the table in the screenshot)
image
However in FF I can see it

**Comment by scherler** [comment]
image
Reducing the width by 1% fixes the problem it seems

**Comment by mawinter69** [comment]
The same problem can be seen on the `/computer` page.
Potentially every page, that causes the whole page to scroll is affected. 
E.g. going to https://ci.jenkins.io/job/Plugins/job/configuration-as-code-plugin/job/master/ and
https://ci.jenkins.io/job/Plugins/job/configuration-as-code-plugin/job/master/1605/badge/ also show that behavior.

**Comment by mawinter69** [comment]
Maybe we can make the header and crumbbar sort of sticky so that it is always shown complete regardless of horizontal scroll position.

**Pull request merged**
